test/nn/conv/test_pna_conv.py

```diff
@@ -36,6 +36,16 @@ def test_pna_conv():
         assert torch.allclose(jit(x, adj.t()), out, atol=1e-6)
 
 
+def test_pna_divide_input():
+    x = torch.randn(4, 16)
+    edge_index = torch.tensor([[0, 0, 0, 1, 2, 3], [1, 2, 3, 0, 0, 0]])
+    deg = torch.tensor([0, 3, 0, 1])
+    conv = PNAConv(16, 32, aggregators, scalers, deg=deg, divide_input=True,
+                   train_norm=True)
```

Comment:
We can move `train_norm` test in [nn/aggr/test_scaler.py](https://github.com/pyg-team/pytorch_geometric/blob/master/test/nn/aggr/test_scaler.py)

Reply:
Sure!